src/components/ha-sidebar.html

```diff
@@ -197,6 +201,10 @@
     this._boundUpdateStyles = this.updateStyles.bind(this);
   },
 
+  tr: function (pg, text, options) {
```

Comment:
I think that we should follow https://www.polymer-project.org/1.0/toolbox/localize

However, that is for Polymer 1 and Polymer 2 should be released this month which probably changes things slightly. So I think that we should wait with merging this PR until we have migrated to Polymer 2.